Packages/Models/Sources/Models/Status.swift

```diff
@@ -59,6 +59,8 @@ public struct Status: AnyStatus, Decodable, Identifiable, Equatable, Hashable {
     id + createdAt + (editedAt ?? "")
   }
 
+  public var shouldHighlight: Bool?
```

Comment:
To separate that a bit, I would make a new protocol like protocol `StatusUI` and add this the props inside. Then make `Status` conform to this new protocol. Name the property something like `ui_shouldHighlight`. I know it's not pretty but that way it should be crystal clear what is backend / models status and UI extensions for the app.

Reply:
I like this better than what I have. Will try to come up with a good variable name.